### DISCUSSION TITLE
Installing TheHive

### DISCUSSION THREAD

**Discussion by jaheadley** [Security-Onion-Solutions/securityonion#1904] [Unsupported Versions]
I ran into the stuck at "79% Applying Thehive Salt State" during install, and the only way to finish the installer was to uncheck TheHive. Everything else installed correctly and is working properly. However, I would like to take advantage of the functionality of TheHive. I tried to install it manually, but I ran into errors. Is there a way to use the installer to install just TheHive without erasing anything else? If not, is there a solution to the stuck at "79% Applying Thehive Salt State" problem?

**Comment by TOoSmOotH** [comment]
I would ssh into the box and tail the /root/sosetup.log to see what it is hanging on.

**Comment by jaheadley** [comment]
I did that, but the log didn't tell me anything. This a few weeks ago and I don't remember the specifics. I saw that someone else in the forum had the exact same issue, so I figured it was a known issue of sorts. I could spin up another machine and get the log, but is there anyway to install TheHive after the fact?

**Reply by TOoSmOotH** [reply]
yes but it is not straight forward. I'll see what it takes and we can update the docs.
